mne/viz/misc.py

```diff
@@ -607,7 +607,7 @@ def plot_dipole_amplitudes(dipoles, colors=None, show=True):
         xlim[1] = max(xlim[1], dip.times[-1])
     ax.set_xlim(xlim)
     ax.set_xlabel('Time (sec)')
-    ax.set_ylabel('Amplitude (nAm)')
+    ax.set_ylabel('Amplitude (Am)')
```

Comment:
we should still plot amps in nAm by doing the scaling in the plot like we do for Evoked. That's the good unit for looking at data